Sources/nest/Extensions/ArtifactBundleFetcher+extension.swift

```diff
@@ -0,0 +1,18 @@
+import NestCLI
+import Logging
+
+extension ChecksumOption {
+    init(isSkip: Bool = false, expectedChecksum: String?, logger: Logger) {
+        guard !isSkip else {
+            self = .skip
+            return
+        }
+        guard let expectedChecksum else {
+            self = .printActual { checksum in
+                logger.info("ℹ️ The checksum is \(checksum). Please add it to the nestfile to verify the downloaded file.")
+            }
+            return
+        }
+        self = .needsCheck(expected: expectedChecksum)
```

Comment:
I think `guard` statement is for error case.
```swift
guard foo else {
    // error case
}
// happy path
```

In this function, `skip` and `printActual` cases are not error cases.
So, could you use if statements instead?

Reply:
Thanks!
I've made the corrections here. (https://github.com/mtj0928/nest/pull/36/commits/ff049a9bba8fa8e08633cef1c49581f1e1865698)